packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -275,6 +275,19 @@ class Transforms {
               return Promise.resolve(object);
             }));
 
+          //  Decrypt webUrl
+          promises.push(requestWithRetries(ctx.webex.internal.encryption, ctx.webex.internal.encryption.decryptText,
```

Comment:
Are you happy that `activity.extension.webUrl` will always be present, even in old data? i.e. There is no need for an if statement check here?